MetaMask/metamask-extension
# Bump test-dapp to v7.2.0

## **Description**
Bump test-dapp to v7.2.0
See changes https://github.com/MetaMask/test-dapp/pull/261.

Relates to https://github.com/MetaMask/MetaMask-planning/issues/1264

## Manual Testing Steps

1. Check circle ci runs
2. Locally run tests in your environment yarn test:e2e:chrome
